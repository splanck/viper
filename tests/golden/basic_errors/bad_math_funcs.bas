10 PRINT SQR("x")
