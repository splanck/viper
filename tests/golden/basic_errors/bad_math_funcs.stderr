B2001(.|\n)*\^
